AirCasting/Onboarding/PrivacyOnboarding.swift

```diff
@@ -0,0 +1,106 @@
+// Created by Lunar on 09/06/2021.
+//
+
+import SwiftUI
+
+struct PrivacyOnboarding: View {
+    var completion: () -> Void
+    @State var presentingModal = false
+    var body: some View {
+        VStack(alignment: .leading) {
+            progressBar
+            Spacer()
+            HStack {
+                titleText
+                Spacer()
+            }
+            .padding(.bottom, 20)
+            descriptionText
+            Button(action: {
+                completion()
+            }, label: {
+                Text("Continue")
+            })
+                .buttonStyle(BlueButtonStyle())
+                .padding(.top, 20)
+            Button(action: {
+                presentingModal = true
+            }, label: {
+                Text("Learn More")
+            })
+                .buttonStyle(BlueTextButtonStyle())
+                .sheet(isPresented: $presentingModal) { ModalView(presentedAsModal: self.$presentingModal) }
+        }
+        .padding()
+        .navigationBarHidden(true)
+    }
+
+    struct ModalView: View {
+        @Binding var presentedAsModal: Bool
+        var body: some View {
```

Comment:
Don't be mad for changing the whole body but it's difficult to describe these changes. 🙈
Try it and play with this code, change whatever if you want ;D

```
    struct ModalView: View {
        @Binding var presentedAsModal: Bool
        var body: some View {
            VStack(alignment: .leading, spacing: 20) {
                HStack(alignment: .top) {
                    Spacer()
                    Button(action: { presentedAsModal = false }) {
                        HStack {
                            Image(systemName: "xmark")
                        }
                        .padding()
                    }
                }
                sheetTitle
                sheetDescription
                Spacer()
            }
            .padding()
        }
```